### ISSUE TITLE
Flaky test: test_operator_gpu.test_multi_proposal_op 

### ISSUE THREAD

**Issue by rahul003** [apache/incubator-mxnet#10186]
## Description
test_operator_gpu.test_multi_proposal_op is flaky and fails on CI Windows. I believe there might be nothing specific to the windows environment it failed in. 

## Environment info (Required)

```
What to do:
1. Download the diagnosis script from https://raw.githubusercontent.com/apache/incubator-mxnet/master/tools/diagnose.py
2. Run the script using `python diagnose.py` and paste its output here.

```

Package used (Python/R/Scala/Julia): Python

## Build info (Required if built from source)

Compiler (gcc/clang/mingw/visual studio): gcc

MXNet commit hash: 

Build config:CI Windows GPU

http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/incubator-mxnet/detail/PR-9933/17/pipeline/

## Error Message:
```

======================================================================
FAIL: test_operator_gpu.test_multi_proposal_op
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Anaconda3\envs\py3\lib\site-packages\nose\case.py", line 197, in runTest
    self.test(*self.arg)
  File "C:\jenkins_slave\workspace\ut-python-gpu\tests\python\gpu\../unittest\common.py", line 157, in test_new
    orig_test(*args, **kwargs)
  File "C:\jenkins_slave\workspace\ut-python-gpu\tests\python\gpu\test_operator_gpu.py", line 1746, in test_multi_proposal_op
    check_proposal_consistency(mx.nd.contrib.MultiProposal, 20)
  File "C:\jenkins_slave\workspace\ut-python-gpu\tests\python\gpu\test_operator_gpu.py", line 1742, in check_proposal_consistency
    assert_almost_equal(score_cpu_np, score_gpu_np, atol = 1e-3, rtol = 1e-3)
  File "C:\jenkins_slave\workspace\ut-python-gpu\pkg_vc14_gpu\python\mxnet\test_utils.py", line 493, in assert_almost_equal
    raise AssertionError(msg)
AssertionError: 
Items are not equal:
Error 1.914770 exceeds tolerance rtol=0.001000, atol=0.001000.  Location of maximum error:(7761, 0), a=0.409629, b=0.412333
 a: array([[ 0.99995381],
       [ 0.99990904],
       [ 0.99970675],...
 b: array([[ 0.99995381],
       [ 0.99990904],
       [ 0.99970675],...
-------------------- >> begin captured logging << --------------------
common: INFO: Setting test np/mx/python random seeds, use MXNET_TEST_SEED=1679964676 to reproduce.
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
```
## Steps to reproduce
1.  See CI steps

**Comment by anirudh2290** [comment]
Encountered this again today: http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/incubator-mxnet/detail/PR-10014/6/pipeline

@eric-haibin-lin can you please add flaky test label to this issue

**Comment by zheng-da** [comment]
here as well.
http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/incubator-mxnet/detail/PR-10161/5/pipeline

**Comment by zhreshold** [comment]
Fixed in #11530
